src/server/api/routes/create-batch.router.js

```diff
@@ -12,12 +11,68 @@ const ROLES = require('../../constants/roles');
 // controllers
 const createBatchController = require('../controllers/create-batch.controller');
 
+/**
+ * @swagger
+ * /create-batch:
+ *   post:
+ *     summary: Post values received from add batch form fields
+ *     description:
+ *       Post following values received from add batch form fields&colon crop id,
+ *       customer name, number of seeded pots, start seed date with the userId received from header.
+ *
+ *
+ *        Authentication&#58; <code>super_admin</code> <code>admin</code>
+ *
+ *     tags:
+ *       - Batches
+ *     operationId: createBatchController
+ *     produces: application/json
+ *     parameters:
+ *      - name: authorization
+ *        in: header
+ *        description: Firebase token
+ *        required: true
+ *        default: Bearer ENTER_FIREBASE_TOKEN
+ *        type: string
+ *      - name: cropId
+ *        in: body
+ *        type: integer
+ *        required: true
+ *      - name: userId
+ *        in: body
+ *        type: integer
+ *        required: true
+ *      - name: customerName
+ *        in: body
+ *        type: string
+ *        required: true
+ *      - name: seedPot
+ *        in: body
+ *        type: integer
+ *        required: true
+ *      - name: startSeedDate
+ *        in: body
+ *        type: integer
+ *        required: true
+ *     security:
+ *         firebase_auth:
+ *         - write
+ *         - read
+ *     responses:
+ *       200:
+ *         description: Successful request
+ *       401:
+ *         description: Unauthorized request.
+ *       404:
+ *         description: Not found.
+ *       5XX:
+ *         description: Unexpected error.
+ */
```

Comment:
This one should actually be quite different ❗ , as right now it's not possible to make a successful post request through swagger. [Here's the docs](https://swagger.io/docs/specification/2-0/describing-request-body/) on how to document the parameters in body (see under "Object Payload"). Also, the parameter names must match those inside the controller. I tried it out so here it is to save you some time: 

```
/**
 * @swagger
 * /create-batch:
 *  post:
 *    summary:
 *      Post values received from add batch form fields
 *    description:
 *      Post following values received from add batch form fields&colon crop id,
 *      customer name, number of seeded pots, start seed date with the userId received from header.
 *      Authentication&#58; <code>super_admin</code> <code>admin</code> <code>user</code>
 *    tags:
 *      - Batches
 *    operationId: createBatch
 *    produces: application/json
 *    parameters:
 *      - name: authorization
 *        in: header
 *        description: Firebase token
 *        required: true
 *        default: Bearer ENTER_FIREBASE_TOKEN
 *        type: string
 *      - in: body
 *        name: batch
 *        description: Batch info
 *        schema:
 *          type: object
 *          properties:
 *            fk_crop_id:
 *              type: integer
 *            fk_user_id:
 *              type: integer
 *            customer_name:
 *              type: string
 *            number_of_seeded_pots:
 *              type: integer
 *            seeding_date:
 *              type: string
 *          required:
 *            - fk_crop_id
 *              fk_user_id
 *              customer_name
 *              number_of_seeded_pots
 *              seeding_date
 *    security:
 *      firebase_auth:
 *        - write
 *    responses:
 *      200:
 *        description: Successful request
 *      401:
 *        description: Unauthorized request.
 *      404:
 *        description: Not found.
 *      5XX:
 *        description: Unexpected error.
 */
```

Reply:
I will share this with @hemanth06912 and @gizemcandemir since they have the post endpoints. The code above is just what was on the swagger.json converted to YAML format.
I will review the docs. Thanks for the review!

Reply:
No problem! 😁 P.S. You can still replace it with the working version (I tested it) so that we're finished with that one. 😊👌

Reply:
Thanks @kfazinic :pray: I had it as a commit on my branch in the swagger file, but then I just couldn't get back to this task to add it in the router file for good :see_no_evil: So thanks again 👍:pray:

Reply:
@ricardoaguiar so your PR also closes #151 👍 just wanted to mention here so it will be linked to the issue :)

Reply:
It turns out that we can make a successful post request on swagger. All we needed to add to your code above is `format: date-time` after the type:string for seeding_date. So it looks like this:

`seeding_date:
  type: string
  format: date-time`